### PR TITLE
Issue #36: fix lint test

### LINKED ISSUE #36: nachet-backend lint-test failing
Workflow to deploy was added to the repository which includes lint-test that are falling, I am not sure exactly was causes this but one of the issue seems to be that it's trying to iterate over an object which is not a list

Logs available here
https://github.com/ai-cfia/nachet-backend/actions/runs/7046229033/job/19177590933

### PR DESCRIPTION
**Pull request by RussellJimmies**
#36

### CODE REVIEW

File: azure_storage/azure_storage_api.py [hunk 1]
```diff
@@ -217,6 +217,7 @@ async def get_directories(container_client):
     try:
         directories = {}
         blob_list = container_client.list_blobs()
+        print("PLEASE: ", blob_list)
```
Comment (rngadam) [inline azure_storage/azure_storage_api.py]: debug statement?

File: tests/test_azure_storage_api.py [hunk 2]
```diff
@@ -37,41 +46,43 @@ def test_mount_existing_container(self, MockFromConnectionString):
...
         expected_container_name = "user-{}".format(container_name)
 
+        # Run the test
         loop = asyncio.new_event_loop()
         asyncio.set_event_loop(loop)
-        result = loop.run_until_complete(
-            mount_container(connection_string, container_name, create_container=True)
-        )
+        try:
+            result = loop.run_until_complete(
```
Comment (rngadam) [inline tests/test_azure_storage_api.py]: why are you not using self.loop here?

**Review by rngadam — CHANGES_REQUESTED** [review]

File: azure_storage/azure_storage_api.py (hunk 1 above)
Comment (RussellJimmies) [reply azure_storage/azure_storage_api.py]: It was, left it by mistake.

File: tests/test_azure_storage_api.py (hunk 2 above)
Comment (RussellJimmies) [reply tests/test_azure_storage_api.py]: I mistakenly didn't adapt the tests from the first class.

**Review by rngadam — CHANGES_REQUESTED** [review]
There are significant changes to inference but no tests. There ought to be a test suite here.

@redat97 we also want to measure coverage here to ensure that it doesn't drop below what it was before changes are made.

**Review by rngadam — CHANGES_REQUESTED** [review]
@RussellJimmies the test is failing

**Comment by MaxenceGui** [comment]
New unittest tests for the inference function were added on

- #55 

Therefore I would close the issue since a lots of change happen since this PR was open.

**Pull request closed without merging**